### ISSUE TITLE
Proxy-pallet: Define deactivated & concluded status

### ISSUE THREAD

**Issue by didiermis** [hashed-io/hashed-substrate#208]
In the Adobe XD there are two states for a project that haven't been defined yet:
- Deactivated
- Concluded

![Image](https://user-images.githubusercontent.com/62629075/196196931-bebc3f6d-c18a-45e2-b167-b83c0fb815e6.png)

What validations should we consider for each status? (when a project is deactivated no one can perform any actions on it, a project can't be concluded if it has active drawdowns, etc)

**Comment by AugustoLara** [comment]
I don't think we need those steps for the current Phase. Let's move this to the backlog and remove the options from the UI
